src/ansi-c/check_library.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+for var in "$@"; do
+    cp "${var}" __libcheck.c
+    sed -i 's/__builtin_[^v]/s&/' __libcheck.c
+    sed -i 's/__sync_/s&/' __libcheck.c
+    sed -i 's/__noop/s&/' __libcheck.c
+    cc -std=gnu99 -E -include library/cprover.h -D__CPROVER_bool=_Bool -D__CPROVER_thread_local=__thread -DLIBRARY_CHECK -o __libcheck.i __libcheck.c
+    cc -S -Wall -Werror -pedantic -Wextra -std=gnu99 __libcheck.i -o __libcheck.s -Wno-unused-label -Wno-uninitialized
```

Comment:
Would you mind incorporating the changes that #1243 made to ansi-c/Makefile right away? Otherwise we end up in inconsistency-hell.

Reply:
How soon will #1243 be merged? If the answer is "soon", I'm inclined to wait until it's in and then to rebase this PR. I could then factor the check-library stuff into a script which is called from both the makefiles and the CMake build.

Reply:
#1243 is now merged :-)